notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>This is too much code with no descriptions, I would suggest breaking this up into smaller cells and explaining what is going on in each cell. You have 3 functions, you can split it up into 3 cells and explain each function a little more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Removed Helper function and added some explanations. Thanks